src/scripts/crafttweaker/integrations/mods/galacticcraftAndExtraplanets.zs

```diff
@@ -16,8 +16,10 @@ import mods.GalacticraftTweaker;
 	https://github.com/MJRLegends/GalacticraftTweaker/wiki/CraftTweakerSupport
 */
 function init() {
-	// Remove compressed zinc recipe.
-	GalacticraftTweaker.removeCompressorRecipe(<extraplanets:tier8_items:4>);
```

Comment:
What was the reasoning for this removal to be removed?

Reply:
All the HD plate recipes are <previous_tier> + 5 * <compressed_metal>

For some reason compressed zinc is removed and ingots are used instead. They're not used for anything else, but then neither are some of the other compressed metals from extra planets.

Restoring this recipe is consistent with how other plates are crafted.